parser/ast/misc.go

```diff
@@ -2119,6 +2120,8 @@ type AdminStmt struct {
 	Tables    []*TableName
 	JobIDs    []int64
 	JobNumber int64
+	Limit     int64
```

Comment:
can we use LimitSimple rather then the two fields?

Reply:
Modified.